webpack.config.js

```diff
@@ -245,18 +235,6 @@ module.exports = function(env) {
           to: "theme/login.css"
         }
       ])
-    ]),
-    devServer: {
-      headers: {
-        "Access-Control-Allow-Origin": "*"
-      },
```

Comment:
We need to keep this setting to enable development on localhost.

When Troposphere runs (and serves) the app on localhost:8080, it makes cross-origin requests to get assets from the webpack-dev-server running on localhost:8000.

If Webpack doesn't provide the `Access-Control-Allow-Origin: *` header, the browser won't serve the assets returned to it.